client/src/util/renderThrottle.ts

```diff
@@ -1,20 +1,17 @@
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-export default function renderThrottle(callback: any) {
+export default function renderThrottle<T>(
+  callback: (this: T) => void
+): (this: T) => void {
   /*
   This wraps a call to requestAnimationFrame(), enforcing a single
   render callback at any given time (ie, you can call this any number
   of times, and it will coallesce multiple inter-frame calls into a
   single render).
   */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-  let rafCurrentlyInProgress: any = null;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-  return function f(this: any) {
-    if (rafCurrentlyInProgress) return;
-    // eslint-disable-next-line @typescript-eslint/no-this-alias --- FIXME: disabled temporarily on migrate to TS.
-    const context = this;
+  let rafCurrentlyInProgress: number | null = null;
+  return function f(this: T) {
+    if (rafCurrentlyInProgress) return; // eslint-disable-next-line @typescript-eslint/no-this-alias --- required for functionality
     rafCurrentlyInProgress = window.requestAnimationFrame(() => {
-      callback.apply(context);
+      callback.call<T, never, void>(this);
```

Comment:
Keeping `apply()` would do, since we're not passing arguments, thus no difference between using `apply()` vs. `call()`?

```suggestion
      callback.apply<T, void>(this);
```